oauth2_provider/migrations/0004_application_allowed_scopes.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10b1 on 2016-06-29 15:18
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Should there also be a data migration that sets the scopes for existing applications?